static/js/biomedical/protein/chart.ts

```diff
@@ -341,7 +341,7 @@ function dragNode(
  * Draws the legend for the tissue score chart if the data exists.
  * @param id - div id for the tissue legend
  * @param data - json formatted protein data
- * @returns - empty for null data
+ * @returns empty for null data
```

Comment:
looking at the function definition, this function shouldn't return anything? since the return type is `void`? If nothing is being `return`ed, don't need this line. Here and the rest of the functions in this file

Reply:
Done!